mne/report.py

```diff
@@ -765,7 +765,7 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             # on some version mayavi.core won't be exposed unless ...
             from mayavi import mlab  # noqa, mlab imported
             import mayavi
-        except ImportError:
+        except:  # on some systems importing Mayavi raises SystemExit (!)
```

Comment:
The point of enumerating the exceptions is to catch a specific subset. In this context, we actually don't want to do that -- we want to catch any possible type of exception.